clusterloader2/testing/node-throughput/config.yaml

```diff
@@ -7,9 +7,17 @@
 {{$CONTAINER_IMAGE := DefaultParam .CONTAINER_IMAGE "k8s.gcr.io/pause:3.1"}}
 {{$POD_STARTUP_LATENCY_THRESHOLD := DefaultParam .POD_STARTUP_LATENCY_THRESHOLD "5s"}}
 {{$OPERATION_TIMEOUT := DefaultParam .OPERATION_TIMEOUT "15m"}}
+{{$IS_SMALL_CLUSTER := lt .Nodes 100}}
+{{$NODES_PER_NAMESPACE := MinInt .Nodes (DefaultParam .NODES_PER_NAMESPACE 100)}}
+
+#variables
+{{$namespaces := DivideInt .Nodes $NODES_PER_NAMESPACE}}
+
+{{$schedulerThroughputNamespaces := IfThenElse $IS_SMALL_CLUSTER 1 $schedulerThroughputNamespaces}}
 
 name: node-throughput
-automanagedNamespaces: {{$POD_COUNT}}
+namespace:
+  number: {{AddInt $namespaces $schedulerThroughputNamespaces}}
```

Comment:
It should be just "number: {{$POD_COUNT}}".
Other things you copied from load/config.yaml are not necessary ;)